src/app/core/basic-datatypes/date/date-range-filter/date-range-filter-panel/date-range-filter-panel.component.scss

```
@@ -1,3 +1,5 @@
+@use "../../../../common-components/special-option-labels.scss";
```

Stylelint (17.10.0)

[error] Unexpected extension ".scss" in `@use` (scss/load-partial-extension)

(scss/load-partial-extension)